### DISCUSSION TITLE
Spancat with a custom transformer from pretrained

### DISCUSSION THREAD

**Discussion by martyna-spyra** [explosion/spaCy#13311] [Help: Coding & Implementations]
Hi, I'm getting decent results on span categorisation task using this [transformer+spancat pipeline](https://github.com/explosion/spaCy/discussions/10161). Now I'd like to customise a pretrained transformer model (deberta-v3-base) by increasing the number of hidden layers, such as in the example [here](https://github.com/huggingface/transformers/issues/5816#issuecomment-659620360). What would be the best approach within the Spacy config system? The curated transformers library or a Thinc wrapper for pytorch? What would the implementation of the latter look like for a pretrained model with an additional dropout layer?
